api/apps/api/test/scenarios/start-scenario-calibration.fixtures.ts

```diff
@@ -0,0 +1,104 @@
+import { bootstrapApplication } from '../utils/api-application';
+import { GivenUserIsLoggedIn } from '../steps/given-user-is-logged-in';
+import { OrganizationsTestUtils } from '../utils/organizations.test.utils';
+import { E2E_CONFIG } from '../e2e.config';
+import { ProjectsTestUtils } from '../utils/projects.test.utils';
+import * as request from 'supertest';
+import { HttpStatus } from '@nestjs/common';
+import { ScenariosTestUtils } from '../utils/scenarios.test.utils';
+import { ScenarioType } from '@marxan-api/modules/scenarios/scenario.api.entity';
+
+export const getFixtures = async () => {
+  const app = await bootstrapApplication();
+  const token = await GivenUserIsLoggedIn(app);
+  const organizationId = (
+    await OrganizationsTestUtils.createOrganization(app, token, {
+      ...E2E_CONFIG.organizations.valid.minimal(),
+      name: `Org name ${Date.now()}`,
+    })
+  ).data.id;
+  const response = await ProjectsTestUtils.createProject(app, token, {
+    ...E2E_CONFIG.projects.valid.minimal(),
+    organizationId: organizationId,
+    name: `Project name ${Date.now()}`,
+  });
+  const projectId = response.data.id;
```

Comment:
See `api/apps/api/test/steps/given-project.ts`

Reply:
Usage of `GiveProjectExists` fixture